FutureNorthants/northants-website
# Add archived to file-download

This PR relies on the following:
- https://github.com/FutureNorthants/northants-website/pull/314
- https://github.com/FutureNorthants/northants-design-system/pull/129

## To Test
- Checkout the design system branch, then run `yalc publish`
- Checkout the backend PR and import config
- Checkout this branch on the frontend and `run rm -rf .next/cache && yalc add northants-design-system && yarn install && yarn dev`
- In a new window, run `yarn codegen` and git status to ensure there are no changes to the generated graphql.tsx file
- Visit the page you edited to test the backend PR. The archived file should have the `rel="archived"` attribute and the non archived `rel` should be empty
